bin/i18n/sync-in.rb

```diff
@@ -91,6 +91,23 @@ def get_i18n_strings(level)
       end
     end
 
+    # rubric
+    if level.mini_rubric&.to_bool
+      rubric_properties = Hash.new
+      %w(
+        rubric_key_concept
+        rubric_performance_level_1
+        rubric_performance_level_2
+        rubric_performance_level_3
+        rubric_performance_level_4
+      ).each do |prop|
```

Comment:
We could dynamically generate this list using `level.methods.select{ |m| m.to_s.match(/rubric.*\w$/)}` but I think having a static list like this is better because it is very clear what properties are translatable and because the rubric feature seems pretty stable since no new properties have been added in 3.5 years.